scripts/globals/mobskills/Blizzard_Breath.lua

```diff
@@ -22,4 +22,4 @@ function onMobWeaponSkill(target, mob, skill)
 
     target:delHP(dmg);
     return dmg;
-end;
+end;
```

Comment:
Just so you know in the future there is no need to trim off the end of file newline. I even tend to add them just so I don't see that marker in diffs about it not being present (which is an old carry over from ages past when it used to matter, these days it doesn't). I recommend totally ignoring them.